RELEASE_NOTES.md

```diff
@@ -2,7 +2,7 @@
 
```

Comment:
The commit  `Fix FileWatcher skipping DELETE events` has comments that you might want to remove

Reply:
Done